src/main/java/it/pagopa/pn/deliverypush/legalfacts/LegalFactGeneratorTemplates.java

```diff
@@ -30,6 +31,7 @@
 
 @Slf4j
 @AllArgsConstructor
+@Component
 public class LegalFactGeneratorTemplates implements LegalFactGenerator {
```

Comment:
Non mi è chiaro perchè non aveva component

Reply:
Perchè prima c'era una classe di config (`LegalFactGeneratorConfig`) che istanziava la classe  `LegalFactGeneratorTemplates` o la classe rimossa `LegalFactGeneratorDocComposition` in base ad una configurazione. Dunque per non avere 2 bean dello stesso tipo, venivano lasciate come classi normali e istanziate da una classe di config e non dal contesto Spring.